src/flowTypes.js

```diff
@@ -338,3 +339,19 @@ export type MultiputPart = {
 export type MultiputData = {
     part?: MultiputPart
 };
+
+export type Version = {
```

Comment:
There is a `BoxItemVersion`
Can you check if its duped.

Reply:
It is similar, but it appears the BoxItemVersion has an incomplete type definition and also includes less fields, according to the sample api docs

Reply:
I would refactor and make them the same, you would just need to add ? to properties that are optional and not needed for the object to be partial and just null check them. Should ideally not have two definitions of versions.

Reply:
Combined the types. As far as I could tell from the api docs only 3 properties were mandatory